rfcs/0002-ipfs-governance.md

```diff
@@ -0,0 +1,215 @@
+
+This RFC includes constraints on the size, formation, and sub-division of teams. Those constraints are aimed at encouraging people to work in small teams while maintaining regular communication with other related teams and groups. 
+
+## Self-organizing ways for People to Coordinate Across Teams
+
+When multiple teams have a shared focus or shred need, they can form a Working Group to convene conversations and align work around those focuses or needs.
+
+## Welcoming Entry Points and Clear Funnels for Engagement
+
+Another important goal of these changes is to create welcoming entry points for the community to subscribe to bodies of work and research. These entry points are curated by the Teams and Working Groups themselves. These Teams and Working Groups  should serve the purpose of encouraging new users, engaging with them and enabling contribution as simply as possible.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The IPFS Project  is governed by a collection of _Teams_ and _Working Groups (WGs)_. These Teams and Working Groups are appointed to research, develop and deploy work under their declared scope/focus.
+
+We distinguish between _Teams_ and _Working Groups_.
+- _Teams_ are small groups who work on products (go-ipfs, ipfs-cluster, ipfs-companion) or services (public gateways, testing infrastructure, etc). They have discrete users, use cases, functionality and release roadmaps. A Team must have at least 3 people whose Full Time efforts are primarily focused on that Team's work, and they should avoid getting bigger than 8-10 people
+- _Working Groups_ tackle ongoing, cross-cutting concerns. Some Working Groups focus on shared needs like security, benchmarking or community engagement. Others focus on shared goals and priorities like Enabling Development of Decentralized Applications, or Supporting Decentralized Data Stewardship. Working Groups do not have a size limit, and don't always have people focused full-time on that group's work.
+
+It's common for a person to belong to a single team and multiple Working Groups. Teams are relatively focused with a small range of direct collaborators while Working Groups are cross-team and cross-functional with a larger range of collaborators.  While most of their work revolves around a single Team, the Working Groups allow them to be in conversation with many people from many teams about cross-cutting concerns.  
+
+We want Teams and Working Groups to emerge, function, and dissolve in response to community needs. To accommodate this, Teams and Working Groups are created using the RFC process -- people who want to form a Team or Working Group must propose it in an RFC and get that RFC merged.  That open-ended invitation to create Teams and Working Groups is matched by clear protocols describing when and how working groups should be dissolved.
+
+# Reference-level explanation
```

Comment:
The content in this section seems more like statutes, rather than an "explanation". It's basicly rules, so I'd adapt the title accordingly.